### DISCUSSION TITLE
Icon not showing up in KDE anymore. Is this a known issue?

### DISCUSSION THREAD

**Discussion by SuperSentient** [keepassxreboot/keepassxc#9616] [Q&A] [answered]
I first noticed that the icon for KeepassXC wasn't showing in the Linux Mint 21.1 Cinnamon file manager for KDBX files but no issue on KDE until just recently I'm not sure if it has something to do with the Linux Firmware update or some other update but its not showing the icon for KDBX files on Kubuntu 22.04.2.
Is this a known bug or what?

**Comment by droidmonkey** [comment, marked as answer]
Make sure the .desktop file is still installed in the correct location. Consult your distro documentation or reinstall keepassxc.

**Reply by SuperSentient** [reply]
Is there any harm if the KDBX file not showing an icon?

**Reply by droidmonkey** [reply]
No, that probably means the icon defined in the .desktop file no longer exists.
